### PR TITLE
Add IsHealthy to AdminServer implementation + wire to storage

### PR DESCRIPTION
**Pull request by Martin2112**
Similar to what was done for the log server.

### CODE REVIEW

File: storage/mysql/admin_storage.go [hunk 1]
```diff
@@ -72,6 +72,10 @@ func (s *mysqlAdminStorage) Begin(ctx context.Context) (storage.AdminTX, error)
 	return &adminTX{tx: tx}, nil
 }
 
+func (s *mysqlAdminStorage) CheckDatabaseAccessible(ctx context.Context) error {
```
Comment (codingllama) [inline storage/mysql/admin_storage.go]: nit: Add a simple test for CheckDatabaseAccessible?

File: server/admin/admin_server.go [hunk 2]
```diff
@@ -42,6 +42,11 @@ func New(registry extension.Registry) *Server {
 	return &Server{registry}
 }
 
+// IsHealthy returns nil if the server is healthy, error otherwise.
+func (s *Server) IsHealthy() error {
+	return s.registry.AdminStorage.CheckDatabaseAccessible(context.Background())
```
Comment (codingllama) [inline server/admin/admin_server.go]: I realize the other IsHealthy methods don't take a ctx either, but maybe we should break the consistency here and take a ctx as parameter. WDYT?

**Comment by Martin2112** [comment]
You could have an admin implementation that doesn't share the same storage as the log data. Not sure how likely this is.

File: server/admin/admin_server.go (hunk 2 above)
Comment (Martin2112) [inline server/admin/admin_server.go]: I'll TODO this so they can be all made consistent.

**Comment by Martin2112** [comment]
PTAL.

**Comment by Martin2112** [comment]
I think it got hit by the flake in #617.

**Pull request merged**